python_basics/class02/exercise_6.py

```diff
@@ -57,10 +58,13 @@ def sort_words(hyphen_str):
 	sorted_str: string in a hyphen-separated sequence after 
 	sorting them alphabetically
 	'''
-
-	# code up your solution here
-
-
+	aStr = ''
+	split_words = hyphen_str.split("-")
+	sortedList = sorted(split_words)
+	#print(sortedList)
+	s = '-'
+	return "'" + s.join(sortedList) + "'"
```

Comment:
you don't need any of the quotation marks here - `s.join(sortedList)` will automatically return a string